tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -3659,7 +3659,21 @@ Duration <DURATION>"
 `;
 
 exports[`types_strings_arithmetic.test.w 1`] = `
-"error: Binary operator '+' cannot be applied to operands of type 'num' and 'str'; only (num, num) and (str, str) are supported
+"error: Unexpected 'identifier'
+   --> ../../../examples/tests/invalid/types_strings_arithmetic.test.w:15:5
+   |
+15 | let b += a;
+   |     ^ Unexpected 'identifier'
+
+
+error: Unexpected 'identifier'
+   --> ../../../examples/tests/invalid/types_strings_arithmetic.test.w:15:10
+   |
+15 | let b += a;
+   |          ^ Unexpected 'identifier'
+
+
+error: Binary operator '+' cannot be applied to operands of type 'num' and 'str'; only (num, num) and (str, str) are supported
```

Comment:
this is a good hint that the test isn't doing what we expected it to do

Reply:
Is there an easy way to generate this `hangar/*.snap` files from the CLI? 

is it ``pnpm turbo test`` sufficient?

Reply:
```
cd tools/hangar
turbo test
```
But that still takes a pretty long time. There's probably a quicker way, but I'm not sure, try the dev channel on slack.
You can also just run the compiler on the specific test and see the output:
```
cd libs/wingc
cargo run --example compile -- ../../examples/tests/invalid/something.test.w
```

Reply:
thanks!